massgov/openmass
# DP-32269: Include organization_type in the dataLayer configuration

**Description:**
Changed the dataLayer configuration to output taxonomy terms for the `organization_type` vocab.


**Jira:** (Skip unless you are MA staff)
DP-32269


**To Test:**
Verify that the organization type is output when there is an organization type selected. Ex: `/orgs/executive-office-of-public-safety-and-security`.
Verify that there are no changes to dataLayer when there is no org type set. Ex: `/orgs/massachusetts-parole-board`.


---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)
